SportsGoods.Core/Models/Product.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SportsGoods.Core.Models
+{
+    public class Product
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; } = string.Empty;
+        public string? Description { get; set; } = string.Empty;
+        public string Brand { get; set; } = string.Empty;
+        public double Price { get; set; }
+        public int Quantity { get; set; }
+        public string ProductCategory { get; set; } = string.Empty;
+
+        public static explicit operator Product(Task<Product?> v)
```

Comment:
I think we don't need this one any longer @LVarsaminov

Reply:
Done (removed)